### PR TITLE
added avatar

### PR DESCRIPTION
**Pull request by nishasen**

### CODE REVIEW

File: Stylesheet/css/variables.css [hunk 1]
```diff
@@ -1,11 +1,13 @@
 :root {
-    --st-primary-color: #800000;
-    --st-primary-hover: #CC0000;
-    --st-primary-disabled: #FF5C5C;
+    --st-primary-color: #014465;
```
Comment (coldpigli) [inline Stylesheet/css/variables.css]: Suggestion: To use hsl format for colors instead of hex codes.

File: Stylesheet/Components/avatar.css [hunk 2]
```diff
@@ -1 +1,37 @@
-/* This css file is for avatars */
\ No newline at end of file
+/* This css file is for avatars */
+
+/* This css file is for avatars */
```
Comment (coldpigli) [inline Stylesheet/Components/avatar.css]: Repetition of comment here.

File: Stylesheet/css/variables.css (hunk 1 above)
Comment (nishasen) [reply Stylesheet/css/variables.css]: Okay I will look into it and change, Thank you

File: Stylesheet/css/utilities.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+.text-right {
+    text-align: right;
+}
+
+.text-center {
+    text-align: center;
+}
+
+.centered {
```
Comment (ttahm3d) [inline Stylesheet/css/utilities.css:85]: Why are you using absoulte + transform for centering content?

Better approach would be to use flex box

```
display: flex;
align-items:center;
justify-content: center;
```

File: playground.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Playground</title>
+    <link href="styles.css" rel="stylesheet">
+</head>
+<body>
+    <div class="avatar avatar-lg-size">
+        <img src="Stylesheet/Images/avatar.jpg" class="img-responsive img-round" alt="avatar">
```
Comment (ttahm3d) [inline playground.html]: Move your images to separate folder. Dont include them in stylesheets folder

File: Stylesheet/Components/avatar.css (hunk 2 above)
Comment (nishasen) [reply Stylesheet/Components/avatar.css]: Yes changing that

File: Stylesheet/css/variables.css [hunk 5]
```diff
@@ -1,11 +1,13 @@
 :root {
-    --st-primary-color: #800000;
```
Comment (ttahm3d) [inline Stylesheet/css/variables.css:2]: Any reason why you are using the prefix --st in your variable names?

File: Stylesheet/css/utilities.css (hunk 3 above)
Comment (nishasen) [reply Stylesheet/css/utilities.css:85]: Okay will change this

File: Stylesheet/css/typography.css [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+    font-weight: 600;
+    font-size: 2rem;
+}
+
+h3 {
+    font-family: var(--st-subheader-text);
+    font-weight: 500;
+    font-size: 1.5rem;
+}
+
```
Comment (ttahm3d) [inline Stylesheet/css/typography.css:23]: Generally `p` tag and `h4` tag have same font size. Only differ in font weight. you are giving same font weight to both of them

File: Stylesheet/css/utilities.css (hunk 3 above)
Comment (nishasen) [reply Stylesheet/css/utilities.css:85]: Okay will change this

File: Stylesheet/css/general.css [hunk 7]
```diff
@@ -2,7 +2,7 @@
 html, body {
     margin: 0;
     padding: 0;
-    font-size: 62.5%;
+    font-size: 100%;
 }
 
 .img-responsive {
```
Comment (ttahm3d) [inline Stylesheet/css/general.css:8]: width: 100% makes the image take up the extra space it requires for it to maintain its aspect ration. use max-width: 100% to achieve responsiveness

[read this article for detailed explaination](https://forum.freecodecamp.org/t/what-is-the-difference-between-width-100-and-max-width-100-on-image-element-in-css/409205)

**Review by ttahm3d — COMMENTED** [review]
Have added few comments. Please have a look once. Thx.

File: playground.html (hunk 4 above)
Comment (nishasen) [reply playground.html]: Okay will change that

File: Stylesheet/css/typography.css (hunk 6 above)
Comment (nishasen) [reply Stylesheet/css/typography.css:23]: Okay changing it

**Review by aliamaanansari — APPROVED** [review]

**Comment by aliamaanansari** [comment]
✅

File: Stylesheet/css/variables.css (hunk 5 above)
Comment (nishasen) [reply Stylesheet/css/variables.css:2]: Just to differentiate it from other class variables. For my own convenience

**Review by nishasen — COMMENTED** [review]
Made the changes suggested

**Pull request merged**